java/src/org/openqa/selenium/chromium/ChromiumDriver.java

```diff
@@ -135,10 +135,10 @@ protected ChromiumDriver(CommandExecutor commandExecutor, Capabilities capabilit
 
     this.capabilities = cdpUri.map(uri -> new ImmutableCapabilities(
         new PersistentCapabilities(originalCapabilities)
```

Comment:
The changes in this class can be reverted since they are only whitespace changes